include/ck_tile/ops/gemm/pipeline/gemm_pipeline_agmem_bgmem_creg_compute_v4_policy.hpp

```diff
@@ -0,0 +1,390 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2025, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/gemm/warp/warp_gemm_dispatcher.hpp"
+#include "ck_tile/ops/common/tensor_layout.hpp"
+
+namespace ck_tile {
+// Default policy for GemmPipelineAGmemBGmemCRegV1
+// Default policy class should not be templated, put template on member functions instead
+struct GemmPipelineAGmemBGmemCregComputeV4DefaultPolicy
```

Comment:
What about deriving from `UniversalGemmPipelineAgBgCrPolicy` ?
https://github.com/ROCm/composable_kernel/blob/b2c7d7747e93dbb5c613aa8880f9fc942dd5e44a/include/ck_tile/ops/gemm/pipeline/gemm_universal_pipeline_ag_bg_cr_policy.hpp#L13
Most of the code is the same and you could hide base class functions with new implementation where needed.